examples/missing_values.py

```diff
@@ -44,7 +45,7 @@
 
 # Add missing values in 75% of the lines
 missing_rate = 0.75
-n_missing_samples = np.floor(n_samples * missing_rate)
+n_missing_samples = int(np.floor(n_samples * missing_rate))
```

Comment:
You could just do `int(n_samples * missing_rate)` seeing as these are all nonnegative.